docs/deploying.md

```diff
@@ -0,0 +1,98 @@
+        run: npm run deploy -- --message "$(git log -1 --pretty=%s)"
+        env:
+          # Authentication information. See below for how to set this up.
+          OBSERVABLE_TOKEN: ${{ secrets.OBSERVABLE_TOKEN }}
+```
+
+When deploying automatically, you won't be able to login with a browser the way you did for manual deploys. Instead you can authenticate via the environment variable `OBSERVABLE_TOKEN` using an API key from Observable.
+
+To create an API token, on https://observablehq.com open your workspace's settings and choose "API keys". From there create a new API key, and assign it the "Deploy new versions of projects" scope. We'll need to secure give our workflow file access that key. 
+
+To do that, we'll use GitHub secrets. Sign in to the settings of your GitHub, and add a secret named `OBSERVABLE_TOKEN`. You can see [GitHub's documentation](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions) for more information about secrets.
+
+This `deploy.yml` will automatically build and deploy your project whenever you change the code, and once per day to keep your data up-to-date.
+
+### Caching
+
+If you have data loaders that take a long time to run or that simply don't need to update every time you change your project code, you can set up caching to automatically re-use existing data from previous builds. To do that, add the following steps to your `deploy.yml` before you run `build`:
+
+```yaml
+jobs:
+  deploy:
+    steps:
+      # ...
+      - id: date
+        run: echo "date=$(TZ=America/Los_Angeles date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+      - id: cache-data
+        uses: actions/cache@v3
+        with:
+          path: docs/.observablehq/cache
+          key: data-${{ hashFiles('docs/data/*') }}-${{ steps.date.outputs.date }}
+      - if: steps.cache-data.outputs.cache-hit == 'true'
+        run: find docs/.observablehq/cache -type f -exec touch {} +
+      # ...
+```
+
+This uses one cache per calendar day. If you re-deploy multiple times in a day, the results of your data loaders will be-reused on the second and subsequent runs. You can customize the `date` and `cache-data` steps to change the cadence of the caching. For example you could use `date +'%Y-%U'` to cache data for a week or `date +'%Y-%m-%dT%H` to cache it for only an hour.
+
+<div class="note">You'll need to update the paths used in this config if you've customized `observablehq.config.js` to have a different `root`.</div>
+
+## Deploying to other services
+
+The output of Observable Framework is set of static files that can be hosted by many services. To build a hostable copy of your project, run:
+
+```sh
+$ npm run build
+```
+
+Then you can upload the contents of your `dist/` directory to your static webhost of choice. You can customize the output directory using the `output` option of your configuration file. For more options related to deploying to other hosts, see [configuration](./config).
```

Comment:
maybe mention the cleanUrls option?

Reply:
Sorry, I just hadn't pushed my changes. I was using the "resolved" state as a checklist while I was working.